web/src/app/pages/create-survey/create-survey.component.spec.ts

```diff
@@ -277,37 +295,33 @@ describe('CreateSurveyComponent', () => {
       }));
     });
 
-    describe('when active survey has a job', () => {
-      beforeEach(fakeAsync(() => {
-        surveyId$.next(surveyId);
-        activeSurvey$.next(surveyWithJob);
-        tick();
-        fixture.detectChanges();
-      }));
-
-      it('updates the first job after clicking continue', fakeAsync(() => {
-        const name = 'new job name';
-        const jobDetailsComponent = component.jobDetails!;
-        jobDetailsComponent.formGroup.controls[
-          jobDetailsComponent.nameControlKey
-        ].setValue(name);
-        clickContinueButton(fixture);
-        flush();
-
-        expect(jobServiceSpy.addOrUpdateJob).toHaveBeenCalledOnceWith(
-          surveyId,
-          job.copyWith({name})
-        );
-      }));
-    });
-
     it('goes back to survey details component after back button is clicked', () => {
       clickBackButton(fixture);
 
       expect(component.surveyDetails).not.toBeUndefined();
       expect(component.jobDetails).toBeUndefined();
     });
   });
+
+  describe('Review', () => {
+    beforeEach(fakeAsync(() => {
+      component.setupPhase = SetupPhase.REVIEW;
+      surveyId$.next(surveyId);
+      activeSurvey$.next(surveyWithJob);
+      tick();
+      fixture.detectChanges();
+    }));
+
+    it('goes back to job details component after back button is clicked', () => {
+      clickBackButton(fixture);
+
+      expect(component.setupPhase).toBe(SetupPhase.JOB_DETAILS);
+
+      // TODO: figure out why these are not suceeding
```

Comment:
qq: have you tried `tick()` & `flush()` after importing those dialog dependencies into this test file?

Reply:
I tried both and the test still fails